server/keyshare/keyshareserver/conf.go

```diff
@@ -0,0 +1,163 @@
+package keyshareserver
+
+import (
+	"encoding/binary"
+	"html/template"
+	"io/ioutil"
+	"strings"
+
+	irma "github.com/privacybydesign/irmago"
+	"github.com/privacybydesign/irmago/internal/common"
+	"github.com/privacybydesign/irmago/internal/keysharecore"
+	"github.com/privacybydesign/irmago/server/keyshare"
+
+	"github.com/dgrijalva/jwt-go"
+	"github.com/go-errors/errors"
+	"github.com/privacybydesign/irmago/server"
+)
+
+type DBType string
+
+var errUnknownDBType = errors.New("Unknown database type")
+
+const (
+	DBTypeMemory   = "memory"
```

Comment:
Its most important use is I think in `internal/testkeyshare/testkeyshare.go`, which contains a helper function starting a keyshare server. This function is used by some of the unit/integration tests. This means that those tests that perform sessions involving the keyshare server, can now run without having to connect to a postgres database. That is, they run even when running `go test --tags=local_tests`, which excludes all tests that connect to external services such as postgres. That will likely vastly increase test coverage in that case, because there is rather a lot of keyshare code even outside of the keyshare server. On production (nearly) all sessions use a keyshare server, so having that extra coverage adds great value.